src/ucs/datastruct/ptr_map.inl

```diff
@@ -149,9 +150,10 @@ ucs_ptr_map_get(ucs_ptr_map_t *map, ucs_ptr_map_key_t key, int extract,
  * @param [in]  map     Container.
  * @param [in]  key     Key to object pointer.
  *
- * @return       - UCS_OK on success
- *               - UCS_ERR_NO_ELEM if the key is not found in the internal hash
- *                 table.
+ * @return - UCS_OK on success
+ *         - UCS_ERR_NO_PROGRESS if this key is direct and no additional actions
+ *           are required from an upper-layer protocol
```

Comment:
pls document in terms of UCS and not UCP
for example:
UCS_ERR_NO_PROGRESS if this key is direct and therefore no action was performed
(also for other funcs)

Reply:
But we don't do any actions in UCS in case of indirect key. They are performed on an upper-layer (e.g. UCP)

Reply:
that's right.. the return value means no action has been done by ptr_map. and that's it. no need to mention how "upper layer" should /should not handle it.

Reply:
fixed